src/PWRetriever.cpp

```diff
@@ -78,7 +78,7 @@ bool PWRetriever::PWArgsWrapper::_validate_object(const ddwaf_object& input, uin
                 }
                 else
                 {
-                    DDWAF_TRACE("Performing recursive validation of item #%zu", i);
+                    DDWAF_TRACE("Performing recursive validation of item #%llu", i);
```

Comment:
`PRIu64` is the portable macro to use, though in practice I don't think any compiler makes `sizeof(unsigned long long) == 16`